code/M3D/m3d.h

```diff
@@ -2199,6 +2202,9 @@ void _m3d_getpr(m3d_t *model, _unused m3dread_t readfilecb, _unused  m3dfree_t f
     }
     if(freecb && buff) (*freecb)(buff);
 #else
+    (void)readfilecb;
```

Comment:
Same here!

Reply:
Uhm, same what? It is a way to tell the compiler that the argument is not used if you haven't configured the interpreter with the define. Not an issue with gcc and clang which understand __attribute__, and that's the reason why _unused is defined above for MSVC.

Reply:
Maybe just a typo?

Reply:
This is for compilers that don't understand the attribute nor the pragma. Casting to void without using the result is the last resort to silence the unused argument warning. Note that casting without using the result has no side effects, and it was used commonly in early C compilers.